contracts/ExampleNFT.cdc

```diff
@@ -134,7 +134,7 @@ access(all) contract ExampleNFT: NonFungibleToken {
     access(all) resource Collection: NonFungibleToken.Collection, ExampleNFTCollectionPublic {
         /// dictionary of NFT conforming tokens
         /// NFT is a resource type with an `UInt64` ID field
-        access(contract) var ownedNFTs: @{UInt64: {NonFungibleToken.NFT}}
+        access(all) var ownedNFTs: @{UInt64: {NonFungibleToken.NFT}}
```

Comment:
Might be worth adding a note in this comment regarding access control since the interface is `access(contract)`. I didn't realize subtypes could define less restrictive access